src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -8,23 +8,43 @@ import java.nio.charset.Charset
 
 class Lambda {
 
-  def process(inputStream: InputStream, outputStream: OutputStream) = {
-    val apiUser = createUser(lambdaConfig.consignmentApiUser)
+  def process(inputStream: InputStream, outputStream: OutputStream): Unit = {
+    createUsers(lambdaConfig.databaseName)
+
+    outputStream.write("Users created successfully".getBytes(Charset.defaultCharset()))
+  }
+
+  def createUsers(databaseName: String): Boolean = {
+    databaseName match {
+      case "consignmentapi" => createConsignmentApiUsers
+      case "keycloak" => createKeycloakUsers
+    }
+  }
+
+  def createKeycloakUsers: Boolean = {
```

Comment:
Should this be `createKeycloakUser` instead of plural as only one Keycloak user is being created?

Reply:
Good point, I've changed it.